types/models.d.ts

```diff
@@ -217,7 +221,7 @@ declare module 'mongoose' {
     >;
 
     /** Creates a new document or documents */
-    create<DocContents = AnyKeys<TRawDocType>>(docs: Array<TRawDocType | DocContents>, options?: SaveOptions): Promise<THydratedDocumentType[]>;
+    create<DocContents = AnyKeys<TRawDocType>>(docs: Array<TRawDocType | DocContents>, options?: CreateOptions): Promise<THydratedDocumentType[]>;
```

Comment:
Need to add TypeScript types to indicate that, if `immediateError: false`, then the return type is `Promise<Array<THydratedDocumentType | Error>>`